docs/source/widget-list/index.rst

```diff
@@ -4,6 +4,12 @@ List of widgets
 
 Here we collect the information about all the widgets available in AWB and provide a simple example of how to use them.
 
+
+Using the widgets usually just involves importing and displaying them.
+For demos, have a look at the jupyter notebooks (`.ipynb` extension) in
+this folder.
```

Comment:
```suggestion
For demos, have a look at the jupyter notebooks (`.ipynb` extension) in this folder.
```
We should probably install a linter, but generally we should follow the "one sentence per line" rule.